code/modules/mob/living/carbon/human/species/necromorph/tripod.dm

```diff
@@ -857,6 +857,4 @@ https://bigmemes.funnyjunk.com/pictures/Long+boi_073bf6_7722185.jpg
 #undef LEAP_REDUCED_COOLDOWN
 #undef TONGUE_EXTEND_TIME
 
-#undef LEFT_ARM_OFFSETS
-#undef RIGHT_ARM_OFFSETS
```

Comment:
as above, don't delete these

Reply:
currently this is removing a definition that is not defined.

Reply:
ah it seems i didnt use defines here